command/override_key.go

```diff
@@ -13,7 +13,7 @@ func overrideKey() *cobra.Command {
 		Short: "Generates an overrideKey to bypass the Florence dataset version validation step when approving a collection",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			now := time.Now()
-			midnight := time.Date(now.Year(), now.Month(), now.Day()+1, 0, 0, 0, 0, time.UTC)
+			midnight := time.Date(now.Year(), now.Month(), now.Day()+1, 0, 0, 0, 0, time.Local)
```

Comment:
The pedant in me says we really should use the "Europe/London" time zone instead as it's florence's time zone you want to match with not your local machine. However as we can only support from the UK and not from a different time zone, this is totally fine.

Reply:
I was debating this. I thought it made sense to calculate the total mins until midnight in the UK but  given the server can be running anywhere it feels like to makes more sense to use UTC.

I have decided to delete this change and update Zebedee to use UTC for this calculation also and I have updated the documentation to state this.